### PR TITLE
[MIG] purchase_order_revision module: migration to 10.0

### PR DESCRIPTION
**Pull request by cubells**
- Update views
- Add README
- Rename files to new standard
- Update tests

cc @Tecnativa

### CODE REVIEW

File: purchase_order_revision/models/purchase_order.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        # 'orm.Model.copy' is called instead of 'self.copy' in order to avoid
+        # 'purchase.order' method to overwrite our values, like name and state
+        defaults = {'name': old_name,
+                    'revision_number': revno,
+                    'active': False,
+                    'state': 'cancel',
+                    'current_revision_id': self.id,
+                    'unrevisioned_name': self.unrevisioned_name,
+                    }
+        old_revision = super(PurchaseOrder, self).copy(default=defaults)
```
Comment (pedrobaeza) [inline purchase_order_revision/models/purchase_order.py:60]: You are not using 'orm.Model.copy' as the above comment says. Is not needed anymore? If so, just remove the comment.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: purchase_order_revision/models/purchase_order.py (hunk 1 above)
Comment (cubells) [reply purchase_order_revision/models/purchase_order.py:60]: Done.

**Review by pedrobaeza — APPROVED** [review]

**Review by rami-wafaie — APPROVED** [review]
Review + Testing: Looks good to me 
:+1:

**Review by antoniocanovas — APPROVED** [review]
Functional revision.
LGTM

**Comment by pedrobaeza** [comment]
@cubells please squash commits per author and I'll merge.

**Comment by cubells** [comment]
@pedrobaeza done!

**Comment by MiquelRForgeFlow** [comment]
The runbot was red when this PR was merged :confused:

**Pull request merged**